src/Identity/UI/src/Microsoft.AspNetCore.Identity.UI.csproj

```
@@ -23,6 +23,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Content Update="**/*.cshtml" Pack="false" />
```

```suggestion
    <Content Update="**\*.cshtml" Pack="false" />
```

Not that it matters, but for consistency with all the other globs.